cypress/integration/visual-regression.spec.js

```diff
@@ -64,9 +64,26 @@ describe('Percy Snapshots', () => {
   })
 
   describe('Complex components', () => {
-    it('Creates the Tables page snapshot', () => {
-      cy.visit('/#Complex%20components/Tables/Desktop')
-      cy.percySnapshot('Tables')
+    describe('Tables', () => {
+      it('Creates the Desktop Tables page snapshot', () => {
+        cy.visit('/#Complex%20components/Tables/Desktop')
+        cy.percySnapshot('Tables Desktop')
+      })
+
+      it('Creates the Mobile Tables page snapshot', () => {
+        cy.visit('/#Complex%20components/Tables/Mobile')
+        cy.percySnapshot('Tables Mobile')
+      })
+
+      it('Creates the Stateless Tables page snapshot', () => {
+        cy.visit('/#Complex%20components/Tables/Stateless')
+        cy.percySnapshot('Tables Stateless')
+      })
+
+      it('Creates the Portion Column Tables page snapshot', () => {
+        cy.visit('/#Complex%20components/Tables/Portion%20Column')
+        cy.percySnapshot('Tables Portion Column')
+      })
```

Comment:
should we continue to add every case like this or does it make sense to make something that visits all pages and stories creating snapshots?

Reply:
I liked it this way, seems good to me. WDYT?

Reply:
:smile: I don't have an opinion right now, but I think this discussion should be moved to its own PR as this fix is quite urgent.